src/main/java/io/seqera/migtool/Helper.java

```diff
@@ -201,21 +201,31 @@ private static void closeWithWarning(Closeable closeable) {
         tryClose(closeable);
     }
 
-    static public String dialectFromUrl(String url) {
+    static public Dialect dialectFromUrl(String url) {
         if( url==null )
             return null;
         String[] parts = url.split(":");
-        return parts.length > 1 ? parts[1] : null;
-    }
-
-    static public String driverFromUrl(String url) {
-        final String dialect = dialectFromUrl(url);
-        if( "mysql".equals(dialect) )
-            return "com.mysql.cj.jdbc.Driver";
-        if( "h2".equals(dialect))
-            return "org.h2.Driver";
-        if( "sqlite".equals(dialect))
-            return "org.sqlite.JDBC";
-        return null;
+        return parts.length > 1 ? Dialect.from(parts[1]) : null;
+    }
+
+    static public Driver driverFromUrl(String url) {
+        final Dialect dialect = dialectFromUrl(url);
+        if( dialect==null ) {
+            return null;
+        }
+        switch (dialect) {
```

Comment:
Instead of the `switch`, maybe we could add a `driver` property in the `Dialect` enum (with its corresponding `getDriver()`/`toDriver` getter) initializing it accordingly in the constructor of the enum.

Reply:
Yes, that can be done. I just didn't want to refactor the entire code yesterday, but I'm ok improving it a bit more :)